cmd/osctl/cmd/cluster.go

```diff
@@ -129,7 +144,7 @@ func create() (err error) {
 		requests[i] = &node.Request{
 			Input:    *input,
 			Image:    nodeImage,
-			Name:     fmt.Sprintf("master-%d", i+1),
+			Name:     fmt.Sprintf("%s_master_%d", clusterName, i+1),
```

Comment:
We should use `-` instead of `_` since `_` isn't allowed in DNS. 
nit: should we do `master-%s-%d` instead?

Reply:
fixed